package.json

```diff
@@ -13,30 +13,27 @@
   "license": "Apache-2.0",
   "author": "Google LLC",
   "bin": {
-    "gapic-generator-typescript": "build/src/start-script.js",
-    "protoc-gen-typescript_gapic": "build/src/cli.js"
+    "gapic-generator-typescript": "build/src/gapic-generator-typescript.js",
+    "protoc-gen-typescript_gapic": "build/src/protoc-plugin.js"
   },
   "files": [
-    "build/src",
-    "pbjs-genfiles",
-    "templates"
+    "build/**/*.js",
+    "build/**/*.d.ts",
+    "build/templates"
   ],
   "scripts": {
-    "baseline": "node build/tools/update-baselines.js",
-    "clean": "gts clean",
-    "codecov": "c8 --reporter=lcov mocha build/test/unit && c8 report",
-    "compile": "tsc -p .",
-    "compile-protos": "pbjs -p protos -p node_modules/google-gax/protos -t static-module -o pbjs-genfiles/plugin.js google/protobuf/compiler/plugin.proto google/api/annotations.proto google/api/field_behavior.proto google/api/resource.proto google/api/client.proto google/longrunning/operations.proto service_config.proto && pbts pbjs-genfiles/plugin.js -o pbjs-genfiles/plugin.d.ts",
+    "baseline": "node bazel-bin/tools/update-baselines.js",
+    "clean": "bazel clean && rm -rf build",
```

Comment:
I'm surprised bazel wouldn't know `build` is a build artifact and clean it up

Reply:
The thing is, `build` is not a build artifact for bazel :) It puts its stuff into `bazel-*` folders that are actually symlinks pointing directly to hell.

```
$ ls -l bazel-*
lrwxr-xr-x  1 fenster  5001  131 May  8 01:58 bazel-bin -> /private/var/tmp/_bazel_fenster/a716da2979244e454b0762cc74541b5d/execroot/gapic_generator_typescript/bazel-out/darwin-fastbuild/bin
lrwxr-xr-x  1 fenster  5001  100 May  8 01:58 bazel-gapic-generator-typescript -> /private/var/tmp/_bazel_fenster/a716da2979244e454b0762cc74541b5d/execroot/gapic_generator_typescript
lrwxr-xr-x  1 fenster  5001  110 May  8 01:58 bazel-out -> /private/var/tmp/_bazel_fenster/a716da2979244e454b0762cc74541b5d/execroot/gapic_generator_typescript/bazel-out
lrwxr-xr-x  1 fenster  5001  136 May  8 01:58 bazel-testlogs -> /private/var/tmp/_bazel_fenster/a716da2979244e454b0762cc74541b5d/execroot/gapic_generator_typescript/bazel-out/darwin-fastbuild/testlogs
```